README.md

```diff
@@ -102,6 +113,13 @@ Once the backup is completed you should see the backup marked as `Completed`.
 
 
 ### Creating a restore from backup
+#### Creating a restore from local backup/snapshot
```

Comment:
lets put limitations as well for restore from local snapshot if any like 'storageclass can't have WaitForFirstConsumer'

Reply:
and `restore from local backup need to be done from same cluster only where local backups are done, unlike restore of remote backups which can be done to any cluster.`

Reply:
ok